app/Main.hs

```diff
@@ -125,11 +137,22 @@ showVersion = do
   up <- getNewerReleaseVersion gitInfo
   either (hPrint stderr) (putStrLn . ("New upstream release: " <>)) up
 
+printKeybindings :: KeybindingPrint -> IO ()
+printKeybindings p = do
+  kb <- showKeybindings p
+  T.putStrLn kb
+  when (p /= IniPrint) $ do
+    (iniExists, ini) <- getSwarmConfigIniFile
+    let iniState = if iniExists then "is" else "can be created"
```

Comment:
If the `config.ini` doesn't already exist, in addition to saying "can be created at..." maybe it could also explain how to create it, i.e. mention `swarm keybindings --ini > ...`  For that matter, why not add a flag like `swarm keybindings --init` which creates a default `config.ini` in the correct location (if one does not already exist)?

Reply:
Yes, I was thinking about it as well. 👍 

Otherwise, if you wanted to add new keybindings, then you would have to use this trick:
```bash
swarm keybindings --ini | tee ~/.config/swarm/config.ini
#                         ^^^ does not truncate the file
```

Reply:
✅